src/statistiek_hub/utils/datetime.py

```diff
@@ -45,13 +46,14 @@ def convert_to_datetime(date: str = None) -> datetime:
                 month if month != 0 else month + 1,
                 day if day != 0 else day + 1,
             )
-
-            return _date
-        except ValueError:
+        except:
             raise ValueError(
-                f"verkeerd datumformat: {date}, toegestane formats zijn {formats_allowed}"
+                f"verkeerd datumformat voor {date}, toegestane formats zijn {formats_allowed}"
             )
 
+    if _date != None:
```

Comment:
Naast het feit dat vergelijkingen met None altijd met `is` gedaan moeten worden, voegt deze if niets toe. Wanneer een functie geen return value heeft, dan is de return value by default None. Dus of je nou de value None returnt, of niets, het resultaat is hetzelfde. Deze if kan dus weg.